### DISCUSSION TITLE
项目拉下来本地跑不起来，是怎么回事，报很奇怪的问题

### DISCUSSION THREAD

**Discussion by shuimunianhua99** [TelsaCoin/TelsaVideo#42] [General]
因为公司最近要做关于短视频的业务，所以在做技术调研，看到您的demo很好奇，所以拉到本地想跑一跑，结果报以下错误：
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'android'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not resolve com.android.tools.build:gradle:4.1.0.
     Required by:
         project :
      > Could not resolve com.android.tools.build:gradle:4.1.0.
         > Could not get resource 'https://maven.aliyun.com/repository/public/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
            > Could not GET 'https://maven.aliyun.com/repository/public/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.android.tools.build:gradle:4.1.0.
         > Could not get resource 'https://maven.aliyun.com/repository/spring/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
            > Could not GET 'https://maven.aliyun.com/repository/spring/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.android.tools.build:gradle:4.1.0.
         > Could not get resource 'https://repo.maven.apache.org/maven2/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
            > Could not GET 'https://repo.maven.apache.org/maven2/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.android.tools.build:gradle:4.1.0.
         > Could not get resource 'https://dl.google.com/dl/android/maven2/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
            > Could not GET 'https://dl.google.com/dl/android/maven2/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.android.tools.build:gradle:4.1.0.
         > Could not get resource 'https://jcenter.bintray.com/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
            > Could not GET 'https://jcenter.bintray.com/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
   > Could not resolve com.google.gms:google-services:4.3.8.
     Required by:
         project :
      > Could not resolve com.google.gms:google-services:4.3.8.
         > Could not get resource 'https://maven.aliyun.com/repository/public/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
            > Could not GET 'https://maven.aliyun.com/repository/public/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.google.gms:google-services:4.3.8.
         > Could not get resource 'https://maven.aliyun.com/repository/spring/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
            > Could not GET 'https://maven.aliyun.com/repository/spring/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.google.gms:google-services:4.3.8.
         > Could not get resource 'https://repo.maven.apache.org/maven2/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
            > Could not GET 'https://repo.maven.apache.org/maven2/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.google.gms:google-services:4.3.8.
         > Could not get resource 'https://dl.google.com/dl/android/maven2/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
            > Could not GET 'https://dl.google.com/dl/android/maven2/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.google.gms:google-services:4.3.8.
         > Could not get resource 'https://jcenter.bintray.com/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
            > Could not GET 'https://jcenter.bintray.com/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
   > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10.
     Required by:
         project :
      > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10.
         > Could not get resource 'https://maven.aliyun.com/repository/public/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
            > Could not GET 'https://maven.aliyun.com/repository/public/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10.
         > Could not get resource 'https://maven.aliyun.com/repository/spring/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
            > Could not GET 'https://maven.aliyun.com/repository/spring/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10.
         > Could not get resource 'https://repo.maven.apache.org/maven2/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
            > Could not GET 'https://repo.maven.apache.org/maven2/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10.
         > Could not get resource 'https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
            > Could not GET 'https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10.
         > Could not get resource 'https://jcenter.bintray.com/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
            > Could not GET 'https://jcenter.bintray.com/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 59s
Running Gradle task 'assembleDebug'...                             61.0s
[!] Gradle threw an error while downloading artifacts from the network. Retrying to download...

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'android'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not resolve com.android.tools.build:gradle:4.1.0.
     Required by:
         project :
      > Could not resolve com.android.tools.build:gradle:4.1.0.
         > Could not get resource 'https://maven.aliyun.com/repository/public/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
            > Could not GET 'https://maven.aliyun.com/repository/public/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.android.tools.build:gradle:4.1.0.
         > Could not get resource 'https://maven.aliyun.com/repository/spring/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
            > Could not GET 'https://maven.aliyun.com/repository/spring/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.android.tools.build:gradle:4.1.0.
         > Could not get resource 'https://repo.maven.apache.org/maven2/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
            > Could not GET 'https://repo.maven.apache.org/maven2/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.android.tools.build:gradle:4.1.0.
         > Could not get resource 'https://dl.google.com/dl/android/maven2/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
            > Could not GET 'https://dl.google.com/dl/android/maven2/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.android.tools.build:gradle:4.1.0.
         > Could not get resource 'https://jcenter.bintray.com/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
            > Could not GET 'https://jcenter.bintray.com/com/android/tools/build/gradle/4.1.0/gradle-4.1.0.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
   > Could not resolve com.google.gms:google-services:4.3.8.
     Required by:
         project :
      > Could not resolve com.google.gms:google-services:4.3.8.
         > Could not get resource 'https://maven.aliyun.com/repository/public/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
            > Could not GET 'https://maven.aliyun.com/repository/public/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.google.gms:google-services:4.3.8.
         > Could not get resource 'https://maven.aliyun.com/repository/spring/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
            > Could not GET 'https://maven.aliyun.com/repository/spring/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.google.gms:google-services:4.3.8.
         > Could not get resource 'https://repo.maven.apache.org/maven2/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
            > Could not GET 'https://repo.maven.apache.org/maven2/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.google.gms:google-services:4.3.8.
         > Could not get resource 'https://dl.google.com/dl/android/maven2/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
            > Could not GET 'https://dl.google.com/dl/android/maven2/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve com.google.gms:google-services:4.3.8.
         > Could not get resource 'https://jcenter.bintray.com/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
            > Could not GET 'https://jcenter.bintray.com/com/google/gms/google-services/4.3.8/google-services-4.3.8.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
   > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10.
     Required by:
         project :
      > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10.
         > Could not get resource 'https://maven.aliyun.com/repository/public/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
            > Could not GET 'https://maven.aliyun.com/repository/public/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10.
         > Could not get resource 'https://maven.aliyun.com/repository/spring/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
            > Could not GET 'https://maven.aliyun.com/repository/spring/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10.
         > Could not get resource 'https://repo.maven.apache.org/maven2/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
            > Could not GET 'https://repo.maven.apache.org/maven2/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10.
         > Could not get resource 'https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
            > Could not GET 'https://dl.google.com/dl/android/maven2/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)
      > Could not resolve org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10.
         > Could not get resource 'https://jcenter.bintray.com/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
            > Could not GET 'https://jcenter.bintray.com/org/jetbrains/kotlin/kotlin-gradle-plugin/1.6.10/kotlin-gradle-plugin-1.6.10.pom'.
               > Connect to 127.0.0.1:1081 [/127.0.0.1] failed: Connection refused (Connection refused)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 53s
Running Gradle task 'assembleDebug'...                             54.4s
[!] Gradle threw an error while downloading artifacts from the network. Retrying to download...
Exception: Gradle task assembleDebug failed with exit code 1
麻烦大佬看看是怎么回事，本地flutter sdk用的是2.10.4，文件下来之后未做任何改动，因为之前项目用过firebase之类的插件，所以未对项目做任何更改。多谢大佬！

**Comment by charliewwdev** [comment]
是因为没有下载到依赖组件的问题

**Comment by charliewwdev** [comment]
我这个里面用了代理 127.0.0.1:1081 是我代理ip
